libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts

```diff
@@ -56,6 +60,31 @@ export class EstateTemplateService extends BaseTemplateApiService {
         400,
       )
     }
+
+    const youngheirs = applicationData.estateMembers.filter(
+      (heir) => kennitala.info(heir.nationalId).age < 18,
+    )
+// Requirements:
+//   Flag if any heir is under 18 years old without an advocate/defender
+//   Unless official division of estate is taking place, then the incoming data need not be validated
+    if (youngheirs.length > 0) {
+      if (youngheirs.some((heir) => !heir.advocate)) {
+        if (
```

Comment:
Let's not trip the linter :>


```suggestion
    // Requirements:
    //   Flag if any heir is under 18 years old without an advocate/defender
    //   Unless official division of estate is taking place, then the incoming data need not be validated
    if (youngheirs.length > 0) {
      if (youngheirs.some((heir) => !heir.advocate)) {
        if (
```